include/alpaka/kernel/TaskKernelCpuOmp2Threads.hpp

```diff
@@ -192,6 +194,30 @@ namespace alpaka
         {
             using type = TIdx;
         };
+
+        //! \brief Specialisation of the class template FunctionAttributes
+        //! \tparam TDim The dimensionality of the accelerator device properties.
+        //! \tparam TIdx The idx type of the accelerator device properties.
+        //! \tparam TKernelFn Kernel function object type.
+        //! \tparam TArgs Kernel function object argument types as a parameter pack.
+        template<typename TDim, typename TIdx, typename TKernelFn, typename... TArgs>
+        struct FunctionAttributes<AccCpuOmp2Threads<TDim, TIdx>, KernelBundle<TKernelFn, TArgs...>>
+        {
+            //! \param kernelBundle Kernel bundeled with it's arguments. The function attributes of this kernel will be
+            //! determined. Max threads per block is one of the attributes.
+            //! \return KernelFunctionAttributes instance. The default version always returns an instance with zero
+            //! fields. For CPU, the field of max threads allowed by the given kernel function for the block is 1.
+            ALPAKA_NO_HOST_ACC_WARNING
+            ALPAKA_FN_HOST_ACC static auto getFunctionAttributes(
+                [[maybe_unused]] KernelBundle<TKernelFn, TArgs...> const& kernelBundle)
+                -> alpaka::KernelFunctionAttributes
+            {
+                alpaka::KernelFunctionAttributes kernelFunctionAttributes;
+                kernelFunctionAttributes.maxThreadsPerBlock = 1u;
```

Comment:
BUG: This is wrong: The number of threads can be `>=1` depending of how the user is setting the environment variable `OMP_NUM_THREADS`
You must query the device properties to get the correct number.

Reply:
I queried device properties for all CPUs.